translations/ca.json

```diff
@@ -2178,6 +2178,7 @@
                         },
                         "generic": {
                             "aspect_ratio": "Relació d'aspecte",
+                            "refresh_rate": "Velocitat d’actualització (segons)",
```

Comment:
Translations are handled in Localize, you should not add them here.

Reply:
This is my first time contributing to HA -- I didn't find any other finds that contained the translations.   When you say "handled in Localize", what do you mean?  Is there a different git repo for this?

Reply:
src\translations\en.json

This is the only place that it needs to be added

Reply:
Ok, I get it now.  Thanks for pointing that out.

Reply:
Please remove the other translation

Reply:
Removed